src/scss/core/_layout.scss

```diff
@@ -12,6 +12,7 @@
 //   Fluid Width classes
 // -------------------------------------
 .w-7-5 { width: 7.5% !important; }
+.w-9-5 { width: 9.5% !important; }
```

Comment:
Note: `w-7-5` is now no longer used anywhere else.

Reply:
now that I'm seeing this.. maybe remove these two oddballs and just use w-10 and make the action name 1 less percentage

Reply:
Ya that makes sense, updated to `w-10` and `23%` => `22%` width on the action/flow name column.